ansible/roles/local_storage/templates/localvolume.yml.j2

```diff
@@ -8,15 +9,16 @@ spec:
   nodeSelector: 
```

Comment:
Silly github won't let me comment on the specific line for `metadata.name` but it needs to be different for each object in this loop otherwise the last object wins.  While testing this, I change it to `lv-{{instance.storageclassname}}`.